source/navitiacommon/navitiacommon/models/bss_configuration.py

```diff
@@ -0,0 +1,70 @@
+# encoding: utf-8
+
+#  Copyright (c) 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import
+from sqlalchemy.dialects.postgresql.json import JSONB
+from navitiacommon.models import db, TimestampMixin
+
+
+class BssConfiguration(db.Model, TimestampMixin):
+    id = db.Column(db.Text, primary_key=True)
+    network = db.Column(db.Text, unique=False, nullable=False)
+    klass = db.Column(db.Text, unique=False, nullable=False)
+    timeout = db.Column(db.Float, nullable=False, default=2)
```

Comment:
`Float` is a bit misleading here as it obfuscates the type (second ?, ms? etc...). using a `db.Time` would resolve ambiguities.

Reply:
Fair point, I will look if it's possible to do something without having to convert everything manually ten times